pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -168,7 +172,14 @@ resources:
   source:
     uri: git@github.com:ONSdigital/census-rm-kubernetes.git
     private_key: ((github.service_account_private_key))
-    paths: [dependencies/*, grafana-dashboards/*, rabbitmq/*, setup-dependencies.sh]
+    paths: [rabbitmq/*, setup-dependencies.sh]
```

Comment:
Would we want to keep `dependencies\*` in the paths here?

Reply:
Good spot, added it back in